src/resin.sbvr

```diff
@@ -90,6 +90,9 @@ Term: build log
 Term: commit
 	Concept Type: Short Text (Type)
 
+Term: contract
```

Comment:
Nit: we generally try to maintain alphabetical order of term declarations to facilitate lookup, so this should go just after `content hash` a few lines below.